iroh/src/baomap/mem.rs

```diff
@@ -335,7 +337,11 @@ impl ReadableStore for Store {
         )
     }
 
-    fn roots(&self) -> Box<dyn Iterator<Item = Hash> + Send + Sync + 'static> {
+    fn tags(&self) -> Box<dyn Iterator<Item = (Bytes, Cid)> + Send + Sync + 'static> {
+        Box::new(std::iter::empty())
```

Comment:
wouldn't it be useful to have this for the in memory version as well?

Reply:
yeah, I did not touch the mem version at all so far.